scripts/wrappers/test_status.py

```diff
@@ -0,0 +1,92 @@
+from .status import (
```

Comment:
You may want to find a better location for this. I placed this here quick and dirty because Python imports are hell. Maybe you should take the time to figure out a unit test concept and including locations for unit tests. More coverage will definitely help to save time over time.